src/test/groovy/org/arquillian/spacelift/gradle/MyOwnDefaultTestChild.groovy

```diff
@@ -0,0 +1,20 @@
+package org.arquillian.spacelift.gradle
+
+import groovy.transform.CompileStatic
+
+/**
+ * Created by tkriz on 22/04/15.
+ */
+@CompileStatic
+class MyOwnDefaultTestChild extends DefaultTest {
+
+    DeferredValue<String> randomString = DeferredValue.of(String.class).from("Hello world!")
+
+    MyOwnDefaultTestChild(String testName, Object parent) {
+        super(testName, parent)
+    }
+
+    MyOwnDefaultTestChild(String testName, DefaultTest other) {
```

Comment:
Since this is a cloning constructor, it should receive MyOwnDefaultTestChild, not DefaultTest, c&p error, I guess.